aspirateur/zam_aspirateur/__main__.py

```diff
@@ -29,78 +25,68 @@
 def main(argv: Optional[List[str]] = None) -> None:
```

Comment:
It doesn't return `None` anymore, maybe a dedicated `SysCode` type to be explicit?